SimPEG/Directives.py

```diff
@@ -233,7 +260,13 @@ def target(self):
         if getattr(self, '_target', None) is None:
             # the factor of 0.5 is because we do phid = 0.5*|| dpred - dobs||^2
             if self.phi_d_star is None:
-                self.phi_d_star = 0.5 * self.survey.nD
+
+                nD = 0
+                for survey in self.survey:
+                    nD += survey.nD
+
+                self.phi_d_star = 0.5 * nD
```

Comment:
this isn't really the behaviour we will want down the road for a joint inversion. Can we add a doctstring with a 
```
... note:: Currently the target misfit is not set up for joint inversions. Get `in touch <https://github.com/simpeg/simpeg/issues/new>`_ if you would like to help with the upgrade! 
```

Reply:
done